OtripleS.Portal.Web/Models/Teachers/Teacher.cs

```diff
@@ -0,0 +1,7 @@
+namespace OtripleS.Portal.Web.Models.Teachers
+{
+    public class Teacher
+    {
+        
+    }
```

Comment:
Where's the properties on the teacher model?
Also don't forget to add a file header

Reply:
I've been down this road in my previous PR
until we finally agreed to leave it empty

Please check this out, and tell me what's next

https://github.com/hassanhabib/OtripleS.Portal/pull/17